### DISCUSSION TITLE
quarto revealjs presentation column layout with spacing

### DISCUSSION THREAD

**Discussion by shannonpileggi** [quarto-dev/quarto-cli#5418] [Q&A] [answered]
Hi! In quarto revealjs presentations, in order to create a slide with two columns with a bit of space in between I have created a middle empty column like this:

```
::: columns
::: {.column width="50%"}
thing 1
:::

::: {.column width="5%"}
:::

::: {.column width="45%"}
thing 2
:::
:::
```

I believe I saw somewhere a better approach using `layout`, but I cannot find the exact solution. Is there a better approach to this?

**Comment by mcanouil** [comment, marked as answer]
### TL;DR

```qmd
::: {layout="[ 0.5, -0.05, 0.45 ]"}
thing 1

thing 2
:::
```

For more complex "element" in columns, use additional divs.

```qmd
::: {layout="[ 0.5, -0.05, 0.45 ]"}

:::: {#first-column}
- thing 1a
- thing 1b

![some figure](figure.png)
::::

:::: {#second-column}
thing 2
::::
:::
```

### Long version with links

I believe you are interested in this particular comments from another thread: https://github.com/quarto-dev/quarto-cli/discussions/4959#discussioncomment-5428660 (there are other Q&A)
There is sparse documentation here and there at the moment in the documentation, see <https://quarto.org/docs/authoring/figures.html#complex-layouts>.

**Reply by shannonpileggi** [reply]
Thank you very much, that is exactly what I needed!
